### ISSUE TITLE
I still get 0.677736 in 3hop MetaQA full-KG

### ISSUE THREAD

**Issue by mili6qm** [malllabiisc/EmbedKGQA#49]
I used your command, your data, KG embedding and the bn weights and I still get 0.678. Can you provided the log file or other advice to help me rise the accuracy?

python main.py --mode train --relation_dim 200 --hidden_dim 256 \
--gpu 2 --freeze 0 --batch_size 128 --validate_every 5 --hops 3 --lr 0.0005 --entdrop 0.1 --reldrop 0.2  --scoredrop 0.2 \
--decay 1.0 --model ComplEx --patience 5 --ls 0.0 --kg_type full

ps: I have used the LSTM

**Comment by apoorvumang** [comment]
Thanks for your interest.

With 3-hop base and proper finetuning we can get ~0.75 accuracy. We need relation matching to get the results reported in the paper since the path is too long.

Please keep posted for the relation matching code to be uploaded.

Thanks
